lib/index.js

```diff
@@ -159,12 +164,11 @@ function getOptions(options) {
     var result = options.result;
     var stats = endStats(result.stats);
 
+    if (cb) {
+      cb.call(options.context, null, result);
+    }
     if (success) {
-      success.call(options.context, {
-        css: result.css,
-        map: result.map,
-        stats: stats
-      });
+      success.call(options.context, result);
     }
```

Comment:
My bad, `var payload` makes perfect sense. But lets try this way:

```javascript
var result = options.result;
var stats = endStats(result.stats);
var payload = {
  css: result.css,
  map: result.map,
  stats: stats
};

delete options.result;   // since this.options will contain unnecessarily extra stuff

if (cb) {
  cb.call(options.context, null, payload);
}
if (success) {
  success.call(options.context, payload);
}
```